Lib/platform.py

```diff
@@ -1344,6 +1349,121 @@ def freedesktop_os_release():
     return _os_release_cache.copy()
 
 
+### musl libc version support
+# These functions were copied and adapted from the packaging module:
+# https://github.com/pypa/packaging/blob/main/src/packaging/_musllinux.py
+# https://github.com/pypa/packaging/blob/main/src/packaging/_elffile.py
+
```

Comment:
Please add a comment which clarified the license terms on this copied part. It seems that the packaging code is dual licensed under the Apache and BSD license, but it's not clear whether the logic actually originated in the package repo or whether it was copied from elsewhere.

Reply:
Looking at the reference that the code states:
https://gist.github.com/lyssdod/f51579ae8d93c8657a5564aefc2ffbca
it seems it was only an inspiration and not a 1:1 copy of the code.

I see the code was written by @uranusjr so hopefully we could get a clearer answer :)

Reply:
The code (for musl) was pretty much rewritten at some point and doesn’t really resemble thge original inspiration anymore. If I remember correctly I was mostly reading the ELF documentation ([man page](https://man7.org/linux/man-pages/man5/elf.5.html) and [this page](https://refspecs.linuxfoundation.org/elf/gabi4+/ch4.eheader.html)) when I was doing it. The code in `_manylinux` resembles the `guess_runtime` code more previously, but that was mostly taken out after the rewrite.